core/message-tags-3.3.md

```diff
@@ -138,13 +136,8 @@ Client-only tags should be treated as untrusted data. They can contain any value
 and are not validated by servers in any way. The server MAY unescape tag values
 after receiving data and escape those values before sending them out.
 
-Some specifications may involve servers accepting client-initiated tags without
-the client-only prefix, they SHOULD define a validation process to be performed
-by the server.
-
-Tags without the client-only prefix that are not specified to be relayed by the
-server or that fail validation MUST be removed by the server before being sent
-with any response to another client.
+Tags without the client-only prefix MUST be removed by the server before being relayed
+with any message to another client.
```

Comment:
I'm OK with empty TAGMSG being allowed, but I don't see why this needs to remove the explicit mention that valid unprefixed tags are allowed to be sent c2s.

We also should add an example of the client sending unprefixed tags that get removed and the server forwarding an empty TAGMSG to another client.